sonar-db/src/main/java/org/sonar/db/metric/MetricDao.java

```diff
@@ -109,14 +104,8 @@ public void insert(DbSession session, MetricDto item, MetricDto... others) {
     return mapper(session).selectAvailableCustomMetricsByComponentUuid(projectUuid);
   }
 
-  public List<MetricDto> selectByIds(final DbSession session, Set<Integer> idsSet) {
-    List<Integer> ids = new ArrayList<>(idsSet);
-    return DatabaseUtils.executeLargeInputs(ids, new Function<List<Integer>, List<MetricDto>>() {
-      @Override
-      public List<MetricDto> apply(@Nonnull List<Integer> ids) {
-        return mapper(session).selectByIds(ids);
-      }
-    });
+  public List<MetricDto> selectByIds(DbSession session, Set<Integer> idsSet) {
+    return executeLargeInputs(new ArrayList<>(idsSet), ids1 -> mapper(session).selectByIds(ids1));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)